webapp/public/javascripts/controllers.js

```diff
@@ -60,4 +63,18 @@ lunchrControllers.controller('RegisterController', ['$scope', '$http', '$state',
                     $scope.errorMessages = data;
                 })
         }
+    }]);
+
+lunchrControllers.controller('UserMatchingController', ['$state', 'socket',
+    function ($state, socket) {
+        socket.emit('match');
```

Comment:
should we have some error checking here in case we cannot reach sever? like a timeout in case, say for example the internet dropped?